pkg/instrumentors/utils/ebpf.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/cilium/ebpf"
+)
+
+const (
+	showVerifierLogEnvVar = "OTEL_GO_AUTO_SHOW_VERIFIER_LOG"
+)
+
+// LoadEbpfObjects loads eBPF objects from the given spec into the given interface.
+// If the environment variable OTEL_GO_AUTO_SHOW_VERIFIER_LOG is set to true, the verifier log will be printed.
+func LoadEbpfObjects(spec *ebpf.CollectionSpec, to interface{}, opts *ebpf.CollectionOptions) error {
+	showVerifierLogs := shouldShowVerifierLogs()
+	if showVerifierLogs {
+		opts.Programs.LogSize = ebpf.DefaultVerifierLogSize * 100
```

Comment:
Why do we need to extend the default size? Should we consider allowing the log size to be adjusted too?

Reply:
Could we add a comment to share what you've put above, maybe as a declare var?